### PR TITLE
[7809] apps/projectcontainers: delete models and references in preparation of

### PR DESCRIPTION
**Pull request by goapunk**
full removal of the projectcontainers app

**Tasks**
- [x] PR name contains story or task reference
- [x] Documentation (docs and inline)
- [x] Tests (including n+1 and django_assert_num_queries where applicable)
- [x] Changelog

### CODE REVIEW

**Review by philli-m — CHANGES_REQUESTED** [review]
Project container is now listed in projects and I can view it, edit view causes an error though

File: meinberlin/apps/ideas/models.py [hunk 1]
```diff
@@ -130,7 +130,9 @@ class AbstractIdea(module_models.Item, Moderateable, ItemBadgesPropertyMixin):
     )
     slug = AutoSlugField(populate_from="name", unique=True)
     name = models.CharField(max_length=120, verbose_name=_("Title"))
-    description = CKEditor5Field(verbose_name=_("Description"))
+    description = CKEditor5Field(
+        verbose_name=_("Description"), config_name="collapsible-image-editor"
+    )
```
Comment (philli-m) [inline meinberlin/apps/ideas/models.py:135]: should this be here?

File: meinberlin/apps/projectcontainers/forms.py [hunk 2]
```diff
@@ -1,101 +0,0 @@
...
-            )
-            .order_by("name")
-        )
-
-    class Meta:
-        model = models.ProjectContainer
-        fields = ["projects"]
-        required_for_project_publish = ["projects"]
-        widgets = {
-            "projects": Select2MultipleWidget,
```
Comment (philli-m) [inline meinberlin/apps/projectcontainers/forms.py:100]: we are phasing the select2 out anyway as it's not accessible but I think this is last instance of the Select2MultipleWidget so we can also delete any styling that goes with is in the select2.scss but can also be totally deleted when we remove them all

Comment (goapunk) [reply meinberlin/apps/projectcontainers/forms.py:100]: it's still used for the categories in all modules which have them and the `map_choose_polygon_with_preset_widget` as far as I understand?

**Review by philli-m — APPROVED** [review]
looking great!
@m4ra I've tested it works but not sure on the code so will wait for you to check it

**Review by m4ra — APPROVED** [review]
LGTM

**Pull request merged**